data/ssystem_major.ini

```diff
@@ -1954,8 +1954,8 @@ tex_map=obs_earth.jpg
 ; orbit_Inclination=90
 ; orbit_MeanLongitude=90
 ; orbit_Period=100000000000
-; 1AU to make earth orbit less distorted
-orbit_SemiMajorAxis=149600000
+; 800.000 km is about twice Lunar orbit size
```

Comment:
Oh please, don't separate thousands with comma or period, it's super confusing.

Reply:
OKOK, it just helped me count the zeros...

Reply:
ISO 31-0, BIPM, IUPAC, AMA, all recommend using (thin) space for this. C++14 and higher uses apostrophes, and Java uses underscores.